_posts/en/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,259 @@
+  [descriptors][topic descriptors].
+
+- **Sparrow wallet 1.7.2 released:**
+  Sparrow's [1.7.2 release][sparrow 1.7.2] adds [taproot][topic taproot]
+  support, [BIP329][] import and export features (see [Newsletter #235][news235
+  bip329]), and additional support for hardware signing devices.
+
+- **Bitcoinex library adds schnorr support:**
+  [Bitcoinex][bitcoinex github] is a Bitcoin utility library for the Elixir functional programming language.
+
+- **Libwally 0.8.8 released:**
+  [Libwally 0.8.8][] adds [BIP340][] tagged hash support, additional sighash
+  support including [BIP118][] ([SIGHASH_ANYPREVOUT][topic SIGHASH_ANYPREVOUT]), and
+  additional [miniscript][topic miniscript], descriptor, and [PSBT][topic psbt] functions.
+
+## Optech Recommends
+
+[BitcoinSearch.xyz][] is a recently-launched search engine for Bitcoin
+technical documentation and discussions.  It was used to quickly find
+several of the sources linked in this newsletter, a vast improvement
+over other more laborious methods we've previously used.  Contributions
+to its [code][bitcoinsearch repos] are welcome.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+FIXME:harding to update Tuesday
+
+- C-Lightning RC
+- btcpay server 1.7.6+
+- bdk 0.27.0
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo], [Core
+Lightning][core lightning repo], [Eclair][eclair repo], [LDK][ldk repo],
+[LND][lnd repo], [libsecp256k1][libsecp256k1 repo], [Hardware Wallet
+Interface (HWI)][hwi repo], [Rust Bitcoin][rust bitcoin repo], [BTCPay
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Core Lightning #5361][] adds experimental support for peer storage
+  backups.  As last mentioned in [Newsletter #147][news147 backups],
+  this allows a node to store a small encrypted backup file for its
+  peers.  If a peer later needs to reconnect, perhaps after losing data,
+  it can request the backup file.  The peer can use a key derived from
+  its wallet seed to decrypt the file and use the contents to recover
+  the latest state of all of its channels.  This can be seen as an
+  enhanced form of [static channel backups][topic static channel
+  backups].  This PR adds support for creating, storing, and retrieving
+  the encrypted backups.  As noted in this PR's commits, the feature
+  hasn't yet been fully specified or adopted by other LN
+  implementations.
+
+- [Core Lightning #5670][] and [#5956][core lightning #5956] makes
+  various updates to its implementation of [dual funding][topic dual
+  funding] based on both recent changes to the [specification][bolts
+  #851][] and comments from interoperability testers.  Additionally, an
```

Comment:
```suggestion
  #851] and comments from interoperability testers.  Additionally, an
```

Reply:
Thanks. Added a separate commit (probably shouldn't be squashed) which adds a new test for this.